tests/serverpod_test_server/test_integration/logging/endpoint_logging_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/builders/log_settings_builder.dart';
+import 'package:serverpod_test_server/test_util/builders/runtime_settings_builder.dart';
+import 'package:serverpod_test_server/test_util/logging_utils.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var client = Client('http://localhost:8080/');
+  late Serverpod server;
+  late Session session;
+
+  setUp(() async {
+    server = IntegrationTestServer.create();
+    await server.start();
+    session = await server.createSession(enableLogging: false);
+  });
+
+  tearDown(() async {
+    await LoggingUtil.clearAllLogs(session);
+    await await session.close();
+    await server.shutdown(exitProcess: false);
+  });
+
+  test(
+      'Given a log settings that enables all logging when calling a noop method then a single log entry is created.',
+      () async {
+    var settings = RuntimeSettingsBuilder().build();
```

Comment:
Discussion: It is not super clear that the default RuntimeSettingsBuilder has all logging enabled by default.
Would it make sense to make that more clear by having a method that does that specifically?

Reply:
maybe not, hmm...